### ISSUE TITLE
 Accessibility issue: ambiguous link text in settings modal

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1074]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
more info link inside settings modal

### Issue description
There is a link at the bottom of the settings modal to the UV github page. The visible label is "more info" and this is what a screenreader will read out. It is impossible to tell what this more info is and where the link leads: neither link text nor context give a hint. Even a user actually looking for a link to UV or their repo may miss this link.

This is actually an issue for ALL users but users who are not navigating by mouse much more so, because mouse user can hover over the mysterious link and see the web address displayed (in most browsers). 
A screenreader or keyboard user will have to use complicated workarounds to find out where this link leads (proably easiest to just click on it). 

### Steps to reproduce
1. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
2. Activate to the settings modal
3. Find the link to the github repo

### Expected behaviour
Link text should be more descriptive, for example "more info about Universal Viewer". 

### Possible fix
Change the visible link text. 
Note that just adding an accessible label without changing the visible text would not fix this issue for everyone, and may even introduce other problems (accessible labels must contain the visible label, see WCAG Criterion 2.5.3 Label in Name) 

### WCAG criterion
2.4.4 Link Purpose (In Context) (Level A)

### Related code

```html
<a href="https://github.com/universalviewer/universalviewer" target="_blank">more info</a>
```

### Other information
**Note**: Although there is an exception in the rule stating this behaviour is acceptable in instances where the link text is ambiguous to all users (not just disabled), I would not apply the exception here because the link is surely not meant to be ambiguous and also because the workaround for sighted users is so much easier and quicker than screenreader users. 
And most of all: Fixing the link text will make UV better for everyone.

**Comment by Geoffsc** [comment]
I would be happy to take this one, thanks

**Comment by Geoffsc** [comment]
@LlGC-szw I need translations for the following string to add to my PR:
"More info about the Universal Viewer"

**Comment by LlGC-szw** [comment]
More info about the Universal Viewer = Mwy o wybodaeth am y Universal Viewer

**Comment by demiankatz** [comment]
> More info about the Universal Viewer = Mwy o wybodaeth am y Universal Viewer

Thanks, @LlGC-szw! Do we have a process for obtaining the other three translations (French, Polish and Swedish)?

**Comment by LlGC-szw** [comment]
We don't so I'll ask on the general forum if there's anyone who could contribute translations for those languages.

**Comment by erinburnand** [comment]
French - En savoir + sur le Universal Viewer (apparently the + is used as plus, or more, in French)

Polish - więcej informacji o Universal Viewer
